src/v0/destinations/sendgrid/config.js

```diff
@@ -38,8 +38,9 @@ const TRACK_EXCLUSION_FIELDS = [
   "trackingSettings"
 ];
 
-const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts`;
+const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts?ids=[IDS]`;
 const MAX_BATCH_SIZE = 30000;
```

Comment:
This is not required, right?

Reply:
`MAX_BATCH_SIZE` is required in tranform.js of sendgrid